src/CMakeLists.txt

```diff
@@ -235,18 +313,9 @@ install(TARGETS CCTag
         INCLUDES DESTINATION "${include_install_dir}")
 
 # Headers:
-
-
-if(CCTAG_WITH_CUDA)
-  install(DIRECTORY "cctag"
```

Comment:
What is the problem with this code?

Reply:
I see in the PR comment:  "install the headers with an explicit list of files, otherwise empty directories may occur (e.g. cuda if cuda is disabled)".
If it's only for the empty folder, it increase the complexity too much for this small gain.

Reply:
also all the test directories were added.
This is not something that can easily avoided with cmake
https://gitlab.kitware.com/cmake/cmake/-/issues/17122
other than using GLOB with all the problems it creates 
https://stackoverflow.com/questions/55451084/cmake-files-matching-pattern-copies-empty-directories/55722518#55722518

And for the record I added this  because vcpkg complains if you install empty directories

Reply:
ok